AnotherSmallFile.txt

```diff
@@ -1,117 +0,0 @@
-Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-Phasellus porta dui non massa pellentesque semper.
-Nunc eget maximus metus.
-Donec laoreet aliquam leo ac tempus.
-Donec nulla ipsum, ultricies ac elit vel, dapibus fringilla neque.
-Proin sit amet consectetur velit.
-Morbi facilisis purus nec pretium mattis.
-Phasellus eu augue commodo, tincidunt nisl quis, porta nisi.
-
-Ut blandit vitae ligula ut auctor.
```

Comment:
Reply from CodeFlow.

---
In reply to: [68511218](https://github.com/kivancmuslu/GitHubTest/pull/1#discussion_r68511218) [](ancestors = 68511218)